client/lib/form-base/actions.js

```diff
@@ -0,0 +1,82 @@
+// SET_FORM_DATA_VALUE is used to update a form field's underlying setting, e.g. selected_payment_method_id
```

Comment:
I don't know how I feel about this file (and the reducers one). If it's specific to a part of the application, I think it should go into that sub-application directory tree. Trying to over-generalize didn't give us good results the last time :)